components/Main.tsx

```diff
@@ -1,50 +1,51 @@
 import Link from "next/link";
-import Navbar from "./Navbar";
+import Navbar from "./NavBar/Navbar";
 
-const Main = () => {
+
+  const Main: React.FC = () => {
+  const isDark = false; 
   return (
-    <div id="home" className="w-full lg:h-screen text-center  ">
-      <Navbar />
-      <div className="max-w-[1240px] w-full h-full mx-auto py-10 flex justify-center items-center ">
+    <div id="home" className="w-full lg:h-screen text-center">
+      <Navbar isDark={isDark} />
+      <div className="max-w-[1240px] w-full h-full mx-auto py-10 flex justify-center items-center">
         <div>
           <h1 className="pt-20 pb-4 text-gray-700">
-            Hi, I'am <span className="text-[#5651e1]">Khaled</span>{" "}
+            Hi, I`m <span className="text-[#5651e1]">Khaled</span>{" "}
           </h1>
-          <h1 className="py-4 text-gray-700">A Front-End Developer</h1>
+          <h1 className="py-4 scale-95 text-gray-700">A Front-End Developer</h1>
           <p className="py-4 text-gray-600 max-w-[70%] m-auto">
-            I'am a passionate trainee web developer specializing in React,
+            I`m a passionate trainee web developer specializing in React,
             Tailwind CSS, GitHub, and Vercel. Skilled in creating engaging web
-            applications with dynamic UIs. Proficient in Reacts component-based
+            applications with dynamic UIs. Proficient in React`s component-based
             architecture, state management, and efficient routing. Experienced
             in utilizing Tailwind CSS for streamlined styling. Actively
             contributing on GitHub and deploying applications with Vercel.
             Seeking new opportunities and collaborations in web development.
-            <span className="text-[#5651e1]">Lets connect!</span>
+            <span className="text-[#5651e1]">Let`s connect!</span>
           </p>
           <div className="flex items-center justify-between max-w-[330px] m-auto py-4">
             <div className="rounded-full shadow-lg shadow-gray-400 p-5 cursor-pointer hover:scale-110 ease-in duration-300">
               <Link href="https://www.linkedin.com/in/khaledHN/">
-                <FaLinkedinIn />
+                <FaLinkedinIn className="text-[#5651e1]" />
               </Link>
             </div>
             <div className="rounded-full shadow-lg shadow-gray-400 p-5 cursor-pointer hover:scale-110 ease-in duration-300">
               <Link href="https://github.com/KhaledHN">
-                <FaGithub />
+                <FaGithub className="text-[#5651e1]" />
               </Link>
             </div>
             <div className="rounded-full shadow-lg shadow-gray-400 p-5 cursor-pointer hover:scale-110 ease-in duration-300">
               <Link href="https://mail.google.com/mail/u/0/#sent?compose=CllgCJqXxhFbpRhxNlVPhLsSwWrlkqhLtXRKFkCtRfxVBnfFHZrnmZNmzdTStkWKskMNjXHPRhg">
-                <AiOutlineMail />
+                <AiOutlineMail className="text-[#5651e1]" />
```

Comment:
How is this related to JS to TS feature?

Reply:
this is UI enhancements